applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_explicit_dynamic_solver.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, model, custom_settings):
         {
             "scheme_type"                : "central_differences",
             "time_step_prediction_level" : 0,
+            "delta_time_refresh"         : 1000,
```

Comment:
Why not 0?
This changes the behavior!

Reply:
True (but it is an expensive calculation and doing once is OK for the whole calculation if the mesh is not very distorted or there is a mass change)

Reply:
Ok for me but decision is up to @KlausBSautter bcs his meshes change a lot

Reply:
this is ok for me ... I am currently checking my tests, thanks for the hint!